python/test_file_generator.py

```diff
@@ -2073,12 +1990,12 @@ def v678_5_file(self):
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v679_1_file(self):
-		"""Set DTI to blank or aa."""
+		"""Set DTI to aa."""
 		name = "v679_1.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.dti = lar.dti.map(lambda x: random.choice(("", "aa")))
+		lar.dti = "1.5"
```

Comment:
Does this value produce failures?

The FIG reads: 1) Debt-to-Income Ratio must be a number, Exempt, or NA, and cannot be left blank.

Confirm it is performing as needed.

Reply:
I set this to "AA" - which should produce formatting errors.